internal/grpc/interceptors/eventsmiddleware/events.go

```diff
@@ -67,6 +67,8 @@ func NewUnary(m map[string]interface{}) (grpc.UnaryServerInterceptor, int, error
 		if ev != nil {
 			if err := events.Publish(publisher, ev); err != nil {
 				log.Error(err)
+			} else {
+				log.Debug("event emitted just fine")
```

Comment:
If we need this log, please also add an identifier which event was emitted fine. Otherwise the log is mostly useless.